src/commands/server/start.ts

```diff
@@ -205,6 +204,7 @@ export default class Start extends Command {
       flags.cheimage && ignoredFlags.push('--cheimage')
       flags.debug && ignoredFlags.push('--debug')
       flags.domain && ignoredFlags.push('--domain')
+      flags.multiuser && ignoredFlags.push('--multiuser')
```

Comment:
Shouldn't chectl flags have higher priority over config file?

Reply:
They have a higher priority over defaults but a less priority in comparison with provided CR